pygmt/tests/test_makecpt.py

```diff
@@ -57,25 +56,25 @@ def test_makecpt_plot_points(points, region):
 
 
 @pytest.mark.mpl_image_compare
-def test_makecpt_plot_grid(grid):
+def test_makecpt_plot_grid(position):
     """
     Use static color palette table to change color of grid.
     """
```

Comment:
The name and docstring don't make sense anymore, because we're not plotting grids here.

Reply:
updated in 8bd56a3